posthog/api/event_definition.py

```diff
@@ -43,9 +52,14 @@ def get_queryset(self):
             else:
                 search = self.request.GET.get("search", None)
                 search_query, search_kwargs = term_search_filter_sql(self.search_fields, search)
+                # Select all columns except `tags` so that queryset doesn't try to fetch one-to-many tags
+                event_definition_fields = ", ".join(
+                    [f.column for f in EnterpriseEventDefinition._meta.get_fields() if f.name != "tags"]  # type: ignore
+                )
+
                 ee_event_definitions = EnterpriseEventDefinition.objects.raw(
                     f"""
-                    SELECT *
+                    SELECT {event_definition_fields}
```

Comment:
1. Good point, adding a comment here.
2. Nope there isn't a need for that because the TaggedItemSerializerMixin abstracts the responsibility for fetching the actual data for the `tags` field.
3. oo good idea